renku/cli/__init__.py

```diff
@@ -196,6 +196,7 @@ def help(ctx):
 cli.add_command(init_command)
 cli.add_command(log)
 cli.add_command(migrate)
+cli.add_command(check_immutable_template_files)
```

Comment:
We should add `hidden=True` in `migrate.py` for `check_immutable_template_files` to hide it from normal help output.

Reply:
👍 
Done